### ISSUE TITLE
Fix transistor documentation

### ISSUE THREAD

**Issue by davidhutchens** [logisim-evolution/logisim-evolution#1783]
Our current transistor documentation does not accurately describe its behavior. The full table of behavior and the "summarized form" do not describe the same behavior. The wording describing the East edge output is also inaccurate.

**Comment by mbaillif** [comment]
Hello,

In parallel to Logsim-evolution, I'm in the process of modifying and adapting the documentation to version 3.9.

The transistor parts and the memory group are already there, and if you'd like to review them, I'd be delighted to do so. 

I'd appreciate it if someone would proofread all the pages with a _V in the document cell docstatus.html and libstatus.html but obviously we can stick to the transistor page. Click on the table cells in docstatus.html or libstatus.html to open the document(s).

github branch: mbaillif/Logisim-evolution-documentation
It's available at: https://mbaillif.github.io/Logisim-evolution-documentation/
or in  www.baillifard.com/logisim

There are still a lot of mistakes to correct, but I'm making progress. That's over 100 hours of work.

What we can do:
Minimal: Push Transistor documents into the logisim-evolution repository.

Better: Push Transistor and memory documents into the logisim-evolution repository.

But in the end, it would be nice to push all the Logisim-evolution-documentation documents into the logisim-evolution repository.  But I'd like someone to go through the documentation first.

**Comment by R3dst0ne** [comment]
A while ago, the idea came up to move the documentation to its own repository, because it would make editing a lot easier and enable us to do things like @mbaillif did. 

Now that LSe has its own organization, it is IMO time to do this step.
Since the doc source will still be needed in this repo, the newly created repository can be included as a [git submodule](https://git-scm.com/book/en/v2/Git-Tools-Submodules).

What are your opinions on this?

**Comment by hewzhew** [comment]
I think the concrete documentation mismatch described here may now be addressed by #2624, which updated the transistor behavior tables and the drain/output description in the wiring library help page.

Could a maintainer please confirm whether anything is still missing for this issue, or whether it can be closed now?

**Comment by davidhutchens** [comment]
No. It is still incorrect. If you look at the tables for the P-type transistor, the small summarized form says the Drain is Source when the Gate is 0. But if you look at the full table, you see that the output is U (which is not 0) when both the Gate and Source are 0.

There is a similar problem with the N-type for both inputs 1.

The small summarized tables do not match the actual behavior of the component.

I recommend removing the small summarized tables altogether. Just leave the full table.

Also the line at the bottom of that section reads "An N-type transistor behaves very similarly to a Controlled Buffer. The primary difference is that a transistor is meant for more basic circuit designs." It is not correct. A single transistor cannot be used as a replacement for a controlled buffer. That line should be removed.

Or perhaps better, replace the above line with something like "The transistor is provided to allow instructors to demonstrate how to build the basic logic gates. In more complex designs, users are encouraged to use the logic gates rather than transistors."

**Comment by hewzhew** [comment]
The remaining issues pointed out by @davidhutchens have now been addressed in #2712, which removed the misleading small summary tables and replaced the controlled-buffer comparison. That PR has been merged.

Could a maintainer please confirm whether #1783 can now be closed?

**Comment by davidhutchens** [comment]
I am closing this as completed by PR #2712.
